code/__DEFINES/skills.dm

```diff
@@ -68,7 +68,7 @@
 #define SKILL_ENGINEER_METAL 	1	//metal barricade repair && c4 use
 #define SKILL_ENGINEER_PLASTEEL 2	//plasteel barricade repair
 #define SKILL_ENGINEER_ENGI 	3	//hacking&&planet engine fixing&&apc building (combat engi)
-#define SKILL_ENGINEER_MT 		4	//Telecomms fixing, faster engine fixing (MT)
+#define SKILL_ENGINEER_ST 		4	//Telecomms fixing, faster engine fixing (ST)
```

Comment:
```suggestion
#define SKILL_ENGINEER_MASTER 		4	//Telecomms fixing, faster engine fixing (ST)
```

`UNSKILLED` and `MASTER` should always be the lowest and highest skill name to avoid confusion and reduce lookup.

Reply:
That's not in the contributing guidelines.